### PR TITLE
use new table

### PR DESCRIPTION
**Pull request by lewisrenfrew**

### CODE REVIEW

**Review by RossStewart — APPROVED** [review]

File: src/Persistence.LinnApps/ServiceDbContext.cs [hunk 1]
```diff
@@ -1201,9 +1201,9 @@ private void QueryPartDataSheetValues(ModelBuilder builder)
 
         private void BuildTqmsCategories(ModelBuilder builder)
         {
-            var e = builder.Entity<TqmsCategory>().ToTable("TQMS_CATEGORIES");
+            var e = builder.Entity<TqmsCategory>().ToTable("PART_TQMS_OVERRIDES");
```
Comment (richardiphillips) [inline src/Persistence.LinnApps/ServiceDbContext.cs:1204]: i'd rename the class  as we will probably need tqmscategory at some point too

Comment (richardiphillips) [reply src/Persistence.LinnApps/ServiceDbContext.cs]: merge away. can do plsql stuff later

File: src/Facade/ResourceBuilders/partTqmsOverridesResourceBuilder.cs [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+    public class partTqmsOverridesResourceBuilder 
+        : IResourceBuilder<IEnumerable<PartTqmsOverride>>
+    {
+        private readonly PartTqmsOverrideResourceBuilder rootProductResourceBuilder 
+            = new PartTqmsOverrideResourceBuilder();
+
+        public IEnumerable<PartTqmsOverrideResource> Build(
+            IEnumerable<PartTqmsOverride> rootProducts)
+        {
+            return rootProducts
```
Comment (richardiphillips) [inline src/Facade/ResourceBuilders/partTqmsOverridesResourceBuilder.cs:20]: not sure it's a root product

**Review by richardiphillips — APPROVED** [review]

File: src/Persistence.LinnApps/ServiceDbContext.cs [hunk 3]
```diff
@@ -129,7 +129,7 @@ public class ServiceDbContext : DbContext
 
         public DbQuery<PartDataSheetValues> PartDataSheetValues { get; set; }
 
-        public DbSet<TqmsCategory> TqmsCategories { get; set; }
+        public DbSet<PartTqmsOverride> TqmsCategories { get; set; }
```
Comment (richardiphillips) [inline src/Persistence.LinnApps/ServiceDbContext.cs:132]: probably should rename this too

**Pull request merged**